util/microservice-sdk/js/examples/consumer/Dockerfile

```diff
@@ -20,8 +20,7 @@ COPY npm-shrinkwrap.json npm-shrinkwrap.json
 
 COPY lib ./lib
 COPY index.js index.js
-COPY examples/back-pressure/sample.js ./app/sample.js
-COPY examples/back-pressure/config.js ./app/config.js
+COPY examples/consumer/sample.js ./app/sample.js
 
```

Comment:
I created the Dockerfile with node-10.16.3-alpine, because the old lib and node-kafka only worked with 10, right?

Reply:
I can test if it works with node 12.x.
There is a newer version of node-rdkafka that we can try to use too.

Reply:
It seems to have some problem with the docker image for node 12, but I din't put some effort to scrutinize it.